docs/roman/source_detection/arguments.rst

```diff
@@ -0,0 +1,55 @@
+Arguments
+=========
+The source detection fitting step has several arguments. These can be specified
+by the user by passing them to the step in a Python session, or setting them
+in a parameter file.
+
+* ``--coverage_mask``: True/False to mask pixels with DO_NOT_USE flags in their
+                       dq array. Default is True.
+* ``--kernel_fwhm``: A parameter for DAOStarFinder: size of Gaussian kernel in
+                     pixels. Default is 2.0.
+* ``--sharplo``: A parameter for DAOStarFinder: lower bound for sharpness.
+                 Default is 0.0.
+* ``--sharphi``: A parameter for DAOStarFinder: upper bound for sharpness.
+                 Default is 1.0.
+* ``--roundlo``: A parameter for DAOStarFinder: lower bound for roundness.
+                 Default is -1.0.
+* ``--roundhi``: A parameter for DAOStarFinder: upper bound for roundness.
+                 Default is 1.0.
```

Comment:
If these parameters have natural scales or centers, it would be nice to mention them.  I imagine that 'roundness' is centered at zero (based on hi/lo); not sure what the scale is (i.e., +1 corresponds to an aspect ratio of 2 or something??).  I really don't have any intuition for sharpness, neither in scale nor center.

Reply:
I will mention that perfectly round == 0.0, but I also couldn't find a clear description of what sharpness means other than: 

'Nearly always, adequate limits on the sharpness and roundness criteria for real stars are 0.2 < sharp <1.0, and — 1.0 < round < +1.0' according to the DAOPhot paper.

Reply:
Some insights into what sharpness is:
- "[...] sharpness provides a measure of the intrinsic size of the object with respect to the PSF" ([Annibali et al. 2008](https://iopscience.iop.org/article/10.1088/0004-6256/135/5/1900))
- "The sharpness statistic compares the central pixel to the mean of the surrounding pixels. If this difference is greater than the originally estimated height of the Gaussian or less than 0.2 the height of the Gaussian (for the default values of SHARPLIM) then the star will be rejected." ([DAOPhot manual](http://astroa.physics.metu.edu.tr/MANUALS/daophot_manual/help_daophot.html))

Reply:
Thanks Clare, Mairan.  Okay, I think I read that as sharpness being (central pix - <surrounding pix>)/central pix.  In which case 1 is a really high upper bound---it's impossible to get higher without having negative pixels, though of course those will be present if there is noise.  Then conversely a perfectly flat image has sharpness = 0 everywhere.  That sounds like a reasonable definition.  There would be no natural central value in that case; it would depend on the PSF and its centering in the pixels.  For the doc, I might just say something like "typical values of sharpness range from 0 (flat) to 1 (delta function)".

Reply:
thanks, added clarification.

Reply:
Usually, sources with sharpness close to 1 are discarded as cosmic rays.